azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -28,3 +28,9 @@ def load_team_help():
     short-summary: Manage service endpoints/service connections
     long-summary: https://docs.microsoft.com/en-us/rest/api/azure/devops/serviceendpoint/endpoints
     """
+
+    helps['devops team'] = """
+    type: group
+    short-summary: Manage teams
+    long-summary: https://docs.microsoft.com/en-us/rest/api/azure/devops/core/teams
```

Comment:
Shouldn't there be atleast some text with the link? e.g. _For More info refer documentation here -_

Reply:
I agree with @atbagga . Also if we are adding the link to documentation, then it should be consistent across other commands as well. We are not adding documentation reference in any other commands except for '"service-endpoint" and "team" group.